applications/FluidDynamicsApplication/python_scripts/compressible_slip_wall_process.py

```diff
@@ -0,0 +1,171 @@
+    return CompressibleSlipWallProcess(model, settings["Parameters"])
+
+
+class CompressibleSlipWallProcess(KratosMultiphysics.Process):
+    """
+    This process helps with convergence by slowly transforming an outlet into a
+    wall. This process has four phases:
+
+    Stages
+    ------
+    - `AWAITING`: Nothing is done before the interval starts.
+    - `RAMP_UP`: The time after interval start. Duration is rampup_period. The
+    wall lets a proportion of the fluid through.
+    - `STEADY` state: Solid free-slip wall. Lasts until the end of the interval.
+    - `FINISHED`: Nothing is done after the interval ends.
+
+    RAMP-UP
+    -------
+    During this stage, the normal component of the momentum or velocity of the
+    selected wall will have exponential decay. After its speed is A times
+    smaller than originally, it extits ramp-up stage.
+
+    This parameter A is hardcoded as `decay_constant`.
+
+    The effect on the normal component of the momentum/velocity `f`, if only
+    manipulated by this process is:
+    ```Python
+           { f0 * A^(-t/period)  if t < period (RAMP-UP)
+    f(t) = {
+           { 0                   if t >= period (STEADY STATE)
+    ```
+    If the momentum/velocity is being affected by any other variable, then the
+    long-term effect is less predictable but each step the momentum/velocity is
+    diminished by a factor of A^(-dt/period).
+
+    STEADY STATE
+    ------------
+    The node is given the SLIP flag, which signals the stategy that it is in
+    charge of applying the boundary condition.
+
+    Key parameters
+    --------------
+    - rampup_period: Duraation of the ramp-up stage.
+    - interval:  The time in this process runs (RAMP_UP + STEADY).
+    - recompute_normals: Enable computing the normals every step.
+    - variable: The variable to control, usually MOMENTUM of VELOCITY
+    """
+
+    decay_constant = 1000
+
+    # Stages
+    AWAITING = 0
+    RAMP_UP = 1
+    STEADY = 2
+    FINISHED = 3
+
+    def __init__(self, model, settings):
+        KratosMultiphysics.Process.__init__(self)
+
+        self.interval = KratosMultiphysics.IntervalUtility(settings)
+
+        settings.ValidateAndAssignDefaults(self.GetDefaultParameters())
+        modelpart_name = settings["model_part_name"].GetString()
+
+        self.model_part = model[modelpart_name]
+        self.rampup_period = settings["rampup_period"].GetDouble()
+        self.rampup_enabled = True
+        self.recompute_normals = settings["recompute_normals"].GetBool()
+        self._stage = self.AWAITING
+        self.variable = KratosMultiphysics.KratosGlobals.GetVariable(settings["variable_name"].GetString())
+
+    def GetDefaultParameters(self):
+        return KratosMultiphysics.Parameters(
+            """
+            {
+                "model_part_name"   : "SPECIFY_MODEL_PART_NAME",
+                "interval"   : [0, "End"],
+                "rampup_period"   : 0,
```

Comment:
```suggestion
                "rampup_time"   : 0,
```
I wouldn't use period (inverse of the frequency). Not long ago we had the same discussion with the `GIDOutputProcess`.